src/common.json/JsonHelpers.cs

```diff
@@ -121,6 +122,7 @@ internal static object ConvertSerializerSettings(BridgeJsonSerializerSettings se
 
         /// <summary>
         /// This contract resolver is used to ignore particular fields when serializing.
+        /// TODO: Should this derive from a camelcase resolver?  Not sure of use cases for this.

```

Comment:
💡☕️ Thanks @markphillips100 for this, I will fyi @hsubramanianaks and @elenavillamil for there eyes around his question and `todo` - so that folks have noticed this question from you.

Reply:
Thanks.  I didn't want to blanket apply the new resolver without knowing for sure the impact.

Reply:
Thanks @Tatsinnit ! Ya I did noticed, I think leaving it as a TODO seems the right step for this PR. Changing this would be larger change with larger impact and since we recently took over this project from a different team I am not sure if there is any historical reasoning behind this.